agent_build_refactored/prepare_agent_filesystem.py

```diff
@@ -0,0 +1,393 @@
+# Copyright 2014-2022 Scalyr Inc.
```

Comment:
Code from this module was mainly extracted from  [agent_build/package_builders.py](https://github.com/scalyr/scalyr-agent-2/blob/master/agent_build/package_builders.py)
Where it was mixed with the builders logic, now it is a separate mudule.